### PR TITLE
Update screencatcher in the e2e folder for avoiding errors

### PR DESCRIPTION
**Pull request by Ohrimenko1988**
Signed-off-by: Ihor Okhrimenko <iokhrime@redhat.com>

### What does this PR do?
Update screencatcher in the e2e folder for avoiding errors

If screenshots store without errors, all works as before
Screenshot from 2019-08-30 13-06-46

If error happens, the error interrupts and stack trace stores in the separated folder and file
Screenshot from 2019-08-30 13-05-33
  

### What issues does this PR fix or reference?
Issue: https://github.com/eclipse/che/issues/14384

### CODE REVIEW

File: e2e/utils/ScreenCatcher.ts
Comment (dmytro-ndp) [inline e2e/utils/ScreenCatcher.ts:46]: Here we can just add `.txt`, or, more obvious, `.error.log` to the **screenshotPath**, to have file name like the follow:
> 0010002--(10:04:38.853): Open workspace.png.error.log

**Comment by dmytro-ndp** [comment]
PR commonly looks good. I would only ask you to wait on results of PR check on CRW Jenkins CI before merging.

**Comment by SkorikSergey** [comment]
crw-ci-test

**Comment by dmytro-ndp** [comment]
@Ohrimenko1988: there is Happy path tests error https://github.com/eclipse/che/pull/14394#issuecomment-526565740: 
> ERROR: /home/e2e/utils/ScreenCatcher.ts[60, 13]: 'errorLog' is declared but its value is never read.

**Comment by dmytro-ndp** [comment]
Oh, you have fixed it recently.

**Review by SkorikSergey — APPROVED** [review]

**Comment by Ohrimenko1988** [comment]
ci-test

**Comment by Ohrimenko1988** [comment]
ci-test

**Pull request merged**
